### PR TITLE
Output plugin to write events to hdfs via webhdfs/httpfs.

### PR DESCRIPTION
**Pull request by dstore-dbap**
Store events in hdfs via webhdfs restapi. This plugin does not have dependencies on any hadoop jars which makes it a bit easier to handle.

### CODE REVIEW

**Comment by wiibaa** [comment]
@suyograo Obsolete PR, 1.5 related discussion in https://github.com/elastic/logstash/issues/3274

**Comment by purbon** [comment]
@wiibaa @suyograo I think we can safely close this now :-)

File: lib/logstash/outputs/webhdfs.rb [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+  # "/user/logstash/dt=%{+YYYY-MM-dd}/%{@source_host}-%{+HH}.log"
+  config :path, :validate => :string, :required => true
+
+  # The format to use when writing events to the file. This value
+  # supports any string and can include %{name} and other dynamic
+  # strings.
+  #
+  # If this setting is omitted, the full json representation of the
+  # event will be written as a single line.
+  config :message_format, :validate => :string
```
Comment (andrewvc) [inline lib/logstash/outputs/webhdfs.rb]: Is there a reason a codec isn't used for message formatting?

Comment (andrewvc) [inline lib/logstash/outputs/webhdfs.rb]: The Line codec seems appropriate as a default: https://github.com/logstash-plugins/logstash-codec-line/blob/master/lib/logstash/codecs/line.rb

Comment (dstore-dbap) [inline lib/logstash/outputs/webhdfs.rb]: Done in https://github.com/dstore-dbap/logstash-output-webhdfs-1/commit/8f1cf04a26eeadf9c448ae7da90af5af8ffee220

**Pull request closed without merging**